tests/BasicRum/Beacon/UserAgentImportTest.php

```diff
@@ -43,10 +45,29 @@ public function testUserAgentSimpleImport()
                     ],
-                ]
-            ));
+                ];
+
+                foreach ($array as $key => $item)
+                {
+                    $tempArray = json_decode($item[1], true);
+                    if( ! isset($tempArray['restiming']))
+                    {
+                        continue;
+                    }
+                    $decompressor = new ResourceTimingDecompressor_v_0_3_4();
+                    $resourceTimingsData = $decompressor->decompressResources(json_decode($tempArray['restiming'], true));
+                    $tempArray['restiming'] = $resourceTimingsData;
+                    $array[$key][1] = json_encode($tempArray);
+                }
+
+                file_put_contents('/home/www/www.tests/boomerangjs/log2.log', print_r($array, true));
```

Comment:
Is there any purpose for writing in a log file to the file system? Please remove it because this doesn't make sense for production systems.

Reply:
No. Of course there is no reason for this in production. Fixed